### ISSUE TITLE
Weird console.logs

### ISSUE THREAD

**Issue by arthureberledev** [dan-kwiat/openai-edge#15]
Hello, following the upgrade to version `1.2.1`, I have noticed some unusual console logs from this package within my application. These logs seem to originate from the path `openai-edge/dist/index.js`.

Here are some samples of these logs:
```
CONSTRUCTOOOOOOR
createChatCompletion 1
createRequestFunction outer
createRequestFunction inner
{ url: '/chat/completions' }
```

**Comment by ryandotelliott** [comment]
These are quite annoying. I cannot seem to find where the would be originating in the repo, are they being created in an external dependency?

**Comment by dan-kwiat** [comment]
@arthureberledev @DeveloperRyan fixed in v1.2.2
